rules/sinks/storages/cookiemanager/java.yaml

```diff
@@ -0,0 +1,7 @@
+sinks:
+  - id: Storages.Android.CookieManager
+    name: Android Cookie Manager
+    patterns:
+      - "(?i)(.*)(set|save)(cookie|token)([s]?)(.*)"
```

Comment:
@gaurav-gogia I see here we are looking for only the methodName, this can result in False positives. 
Better to add some package info as well in regex